README.md

```diff
@@ -1,2 +1,135 @@
-# aaq-core
-Ask-a-question service core service
+<pre align="center" style="text-align:center; font-size: 1vw; background:none;">
+    __            __        ______
+    /""\          /""\      /    " \
+    /    \        /    \    // ____  \
+    /' /\  \      /' /\  \  /  /    )  )
+  //  __'  \    //  __'  \(: (____/ //
+ /   /  \\  \  /   /  \\  \\         \
+ (___/    \___)(___/    \___)\"____/\__\
+ASK-A-QUESTION
+</pre>
+
+<p align="center" style="text-align:center">
+<a href="https://idinsight.github.io/aaq-core/">Docs</a> |
+<a href="#features">Features</a> |
+<a href="#usage">Usage</a> |
+<a href="#architecture">Architecture</a> |
+<a href="#partners">Partners</a>
```

Comment:
This section is TBD?
Should we make it "Funders and Partners" and just add Google.org for now; till Reach feels okay to be included in this list?

Reply:
Good idea!